.gitignore

```diff
@@ -13,3 +13,4 @@ dist
 .node-version
 pages.gen.ts
 test-results
+.test
```

Comment:
What creates this folder?

Reply:
I thought it was `pnpm e2e`, but now I'm not so sure. I'll remove it and add it again only if I see the directory reappear